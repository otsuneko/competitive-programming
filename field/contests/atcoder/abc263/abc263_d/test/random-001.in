10 4 2
7 2 8 9 8 6 6 10 9 8
