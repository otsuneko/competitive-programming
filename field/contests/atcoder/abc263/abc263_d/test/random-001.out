[4, 4, 4, 4, 4, 4, 4, 4, 4, 8]
20
